lib/sippy_cup/scenario.rb

```diff
@@ -158,27 +158,28 @@ def invite(opts = {})
       from_addr = "#{@from_user}@#{@adv_ip}:[local_port]"
       msg = <<-MSG
 
-INVITE sip:#{to_addr} SIP/2.0
-Via: SIP/2.0/[transport] #{@adv_ip}:[local_port];branch=[branch]
-From: "#{@from_user}" <sip:#{from_addr}>;tag=[call_number]
-To: <sip:#{to_addr}>
-Call-ID: [call_id]
-CSeq: [cseq] INVITE
-Contact: <sip:#{from_addr};transport=[transport]>
-Max-Forwards: 100
-User-Agent: #{USER_AGENT}
-Content-Type: application/sdp
-Content-Length: [len]
-#{opts.has_key?(:headers) ? opts.delete(:headers).sub(/\n*\Z/, "\n") : ''}
-v=0
-o=user1 53655765 2353687637 IN IP[local_ip_type] #{@adv_ip}
-s=-
-c=IN IP[media_ip_type] [media_ip]
-t=0 0
-m=audio [media_port] RTP/AVP 0 101
-a=rtpmap:0 PCMU/8000
-a=rtpmap:101 telephone-event/8000
-a=fmtp:101 0-15
+        INVITE sip:#{to_addr} SIP/2.0
```

Comment:
This is all highlighted because it is indented. I don't believe that is valid HEREDOC syntax and will result in the message having spaces before the data on each line resulting in an invalid XML file